testsupport/user_setup.go

```diff
@@ -36,6 +36,7 @@ func CreateMultipleSignups(t *testing.T, awaitilities wait.Awaitilities, targetC
 			Email(fmt.Sprintf("multiple-signup-testuser-%d@test.com", i)).
 			ManuallyApprove().
 			TargetCluster(targetCluster).
+			EnsureMUR().
```

Comment:
We also use `CreateMultipleSignups()` in other tests. And as far as I remember we wanted to wait for usersignups only in [at least ](https://github.com/codeready-toolchain/toolchain-e2e/blob/master/test/e2e/parallel/e2e_test.go#L66). The idea was to start provisioning a bunch of UserSignups and MURs while doing the rest of the tests. Some kind of stress testing since in runtime we can't expect that everything will be provisioned one by one.

So maybe it's better to create another function, something like `CreateMultipleSignupsWithMurs()` which would ensure MURs too:

```
func createMultipleSignups(..., ensureMUR bool) ... {
    ...
 }
 
func CreateMultipleSignups(...) ... {
    return createMultipleSIgnups(false)
}

func CreateMultipleSignupsWithMURs(...) ... {
    return createMultipleSIgnups(true)
}
```

Reply:
@alexeykazakov thank you so much!